tests/rules/test_fix_file.py

```diff
@@ -14,47 +14,47 @@
 a.c:3:1: error: expected expression before '}' token
  }
   ^
-"""),
+"""),  # noqa
 
 ('clang a.c', 'a.c', 3, 1, '',
 """
 a.c:3:1: error: expected expression
 }
 ^
-"""),
+"""),  # noqa
```

Comment:
I considered that alternative as well. However, after noticing that some multiline strings also have trailing whitespace (see 432878bd773319cf034d3070697e8601d4201b6d), I thought it would be easier to just use `# noqa` to ignore both the whitespace errors and the indent errors. My thought process was that it wouldn't be good to edit the string to remove the trailing whitespace, since it was likely pasted from a terminal and we should preserve its formatting. As far as I know, there's no way to have flake8 ignore _just_ the trailing whitespace in multiline strings while still emitting indent errors for them, unfortunately.

Of course, preserving trailing whitespace inside multiline strings may not be all that important (it's hard to imagine a rule that relies on trailing whitespace for matching to work correctly), so maybe we should remove it, remove the `# noqa` comments, and indent the leading triple quotes as you're suggesting. Here's a commit showing what that might look like: https://github.com/josephfrazier/thefuck/commit/25efd1b670c8f9694f6e5617fe4c38666f9b0704. I'm not sure if I like the way [tests/rules/test_fix_file.py](https://github.com/josephfrazier/thefuck/blob/25efd1b670c8f9694f6e5617fe4c38666f9b0704/tests/rules/test_fix_file.py) looks, though :/